frontend/components/Button/button.cy.tsx

```diff
@@ -0,0 +1,285 @@
+import 'material-symbols';
+import '../../styles/global.css';
+import { Button } from './Button';
+
+describe('Button', () => {
+  describe('Primary Button', () => {
+    it('should display primary button', () => {
+      cy.mount(<Button kind="primary" onClick={() => ""}>Test</Button>)
+      cy.get('button').should('have.class', 'bg-blue-500')
```

Comment:
This tests against a certain color and not if the primary button is displayed. When we change the color the primary button is still displayed but the test would fail.